### ISSUE TITLE
Can't convert examples

### ISSUE THREAD

**Issue by sb8244** [apigee-127/swagger-converter#61]
Models with example in the properties or model definition are stripped out. This is a critical piece of valid swagger json that should be preserved. I built a failing spec locally, and looking at how to extend it.

```
"models": {
    "Project": {
      "id": "Project",
      "required": [
        "id"
      ],
      "properties": {
        "id": {
          "type": "integer",
          "description": "Project ID",
          "example": 1
        },
        "createdAt": {
          "type": "date",
          "description": null
        },
        "title": {
          "type": "string",
          "description": null
        }
      },
      "description": "A Project object."
    }
  }
```

or 

```
"models": {
    "Project": {
      "id": "Project",
      "required": [
        "id"
      ],
      "properties": {
        "id": {
          "type": "integer",
          "description": "Project ID",
          "example": 1
        },
        "createdAt": {
          "type": "date",
          "description": null
        },
        "title": {
          "type": "string",
          "description": null
        }
      },
      "description": "A Project object.",
      "example": {
        "id": 1,
        "title": "test"
      }
    }
  }
```

**Comment by sb8244** [comment]
PR addresses this PR

**Comment by IvanGoncharov** [comment]
Fixed in 677b6e0a0b985eaf568b2433da0c10e7f1949893
